client/src/components/global/UserHeader.vue

```diff
@@ -1,7 +1,7 @@
 <template>
   <div class="user-header">
     <div class="problem">
-      <button @click="openModalProblem">[ <span>BETA</span> ] <span>Report a Problem</span></button>
+      <a href="mailto:sender@example.com?subject=New Issue Submission">[ <span>BETA</span> ] <span>Report a Problem</span></a>
```

Comment:
`sender@example.com` should be a configurable email address pulled from somewhere (server-side settings @cpritcha?) or at least hard-coded to something like portmars@asu.edu for now.

Let's also change the subject to "[port-of-mars] issue" or something along those lines

Reply:
I think hard wiring to portmars@asu.edu in the form is fine for now because it only exists there for now. Eventually it would be good to put idea to create some shared settings

Reply:
fixed in newest commit